pandas/tools/merge.py

```diff
@@ -1283,7 +1283,7 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
         argument, unless it is passed, in which case the values will be
         selected (see below). Any None objects will be dropped silently unless
         they are all None in which case a ValueError will be raised
-    axis : {0, 1, ...}, default 0
+    axis : {0/'index'/'rows', 1/'columns'}, default 0
```

Comment:
I would leave out the 'rows'. I think it is more there as a backcompat alias, in most other places we only mention 'index'